qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -311,6 +316,60 @@ def close_tab(self, tab, *, add_undo=True, new_undo=True):
             elif last_close == 'default-page':
                 self.load_url(config.val.url.default_page, newtab=True)
 
+    def _tree_tab_hierarchy_post_remove(self, cur_node):
+        node_parent = cur_node.parent
+
+        if node_parent:
+            node_siblings = list(node_parent.children)
+            node_children = cur_node.children
+
+            if node_children:
+                next_node = node_children[0]
+
+                # prvni node se stane parentem pro ostatní děti
```

Comment:
english please :sweat:

Reply:
Ahah, this somehow survived from the initial patch from someone else. I have no idea what it means. Removed now, thanks

Reply:
"the first node becomes the parent for the other children" according to Google Translate :wink: